docker-compose.dev.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.8"
+services: 
+  client:
+    stdin_open: true
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    ports: 
+      - "3000:3000"
+    volumes:
+      - "/app/node_modules"
+      - "./:/app"
```

Comment:
How do containers in the compose communicate. Is the compose set up to run remotely? how is this done?

Reply:
> How do containers in the compose communicate?

Docker compose is used to define the runtime environment **locally**. In our situation we don't really have any containers communicating with each other because our entire deployment only consists of one container. 

> Is the compose set up to run remotely? how is this done?

Compose files have nothing to do with remote deployments. Those setting are are setup in the google cloud console if you navigate to the cloud run or Kubernetes tab.